packages/core/src/components/forms/_controls.scss

```diff
@@ -285,6 +285,11 @@ $control-indicator-spacing: $pt-grid-size !default;
   $switch-indicator-margin: 2px !default;
   $switch-indicator-size: calc(1em - #{$switch-indicator-margin * 2});
 
+  $switch-indicator-text-font-size: 0.6em; //@KEVINNG MAKE THIS RIGHT
+  $switch-indicator-text-line-height: 1.7em; //@KEVINNG MAKE THIS RIGHT
+  $switch-indicator-text-outside-margin: 0.75em; //@KEVINNG MAKE THIS RIGHT
+  $switch-indicator-text-inside-margin: 1.75em; //@KEVINNG MAKE THIS RIGHT
```

Comment:
changing the font-size on the child element is an unsafe approach here because the checkbox styles rely heavily on `font-size` for all of their layout (see https://github.com/palantir/blueprint/pull/2721). a safer approach would be a grandchild element to isolate `font-size`, like this:

```
.control-indicator
    .control-indicator-child     <-- margins (using em scope from parent)
        .control-indicator-text  <-- font-size
```

Reply:
I wanted to suggest bumping the font weight across to medium to increase readability for the smaller font size. Should that be added under `.control-indicator-text`?

Reply:
i don't think this is the right time to change font-weight.